src/test/java/pl/javastart/euler2/FibonacciCalculatorTest.java

```diff
@@ -0,0 +1,54 @@
+package pl.javastart.euler2;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+class FibonacciCalculatorTest {
+
+    FibonacciCalculator fibonacciCalculator = new FibonacciCalculator();
+
+    int fibonacciSequenceSum;
+
+    @Test
+    void shouldReturn0WhenGiven0() {
+
+        fibonacciSequenceSum = fibonacciCalculator.getFibonacciSequence(0);
+
+        assertThat(0).isEqualTo(fibonacciSequenceSum);
```

Comment:
To deklarujemy w odwrotnej kolejności, powinny być:
`assertThat(fibonacciSequenceSum).isEqualTo(0);`